lib/PrintAsClang/DeclAndTypePrinter.cpp

```diff
@@ -2910,6 +2910,9 @@ static bool isEnumExposableToCxx(const ValueDecl *VD,
       if (auto *params = elementDecl->getParameterList()) {
         for (const auto *param : *params) {
           auto paramType = param->getInterfaceType();
+          // TODO: properly support exporting these optionals. rdar://131112273 
```

Comment:
In general, I prefer that we not put radar numbers into the compiler sources. Source code needs to be fully-describing, and the radar implies that there's more context missing, making it harder for folks who don't have access to the original Radar. Radar numbers are fine in commit messages and test cases, subject to the same constraint that it needs to be clear why the change was made or what the test is testing.

```suggestion
          // TODO: properly support exporting these optionals.  
```

Reply:
Thanks, this makes sense! I applied the change.